Dockerfile

```diff
@@ -31,14 +31,15 @@ RUN /opt/deno.land/v$vDENO/bin/deno compile \
   src/app.ts
 
 RUN source <(./tea -Eds) \
- && /bin/mkdir -p /opt/tea.xyz/v$VERSION/bin \
- && /bin/mv tea /opt/tea.xyz/v$VERSION/bin \
- && cd /opt/tea.xyz && /bin/ln -s v$VERSION 'v*'
+  && /bin/mkdir -p /opt/tea.xyz/v$VERSION/bin \
+  && /bin/mv tea /opt/tea.xyz/v$VERSION/bin \
+  && cd /opt/tea.xyz && /bin/ln -s v$VERSION 'v*'
 
 #------------------------------------------------------------------------------
 FROM debian:buster-slim as stage1
 ARG GITHUB_TOKEN
 ENV GITHUB_TOKEN=$GITHUB_TOKEN
+ENV TEA_PREFIX=/opt
```

Comment:
shouldn't are required. The prefix is determined from `tea`'s install location except when running the installer.